### ISSUE TITLE
JSX linked mirror cursor false trigger location

### ISSUE THREAD

**Issue by zardoy** [microsoft/TypeScript#57127]
### 🔎 Search Terms

typescript react

### 🕗 Version & Regression Information

related: https://github.com/microsoft/TypeScript/issues/56669

### ⏯ Playground Link

https://www.typescriptlang.org/play?jsx=1#code/GYVwdgxgLglg9mABAFQKYGcoAoCUiDeAsAFCJmIBOqUIFSAPACYwBuAfCeV4p92U6wIA6EfgC+YjqT6J6Aembte-BayliSQA

### 💻 Code

```tsx
function Test() {
    return <div>
        
        <div {...{}}>
        </div>
    </div>
}

```
Start writing `<T` on empty ilne

### 🙁 Actual behavior

After triggering linked editing in vscode:
```tsx
function Test() {
    return <div>
        <T
        T<div {...{}}>
        </div>
    </div>
}

```
Linked editing  should not be triggered after `<` in this casetriggering

### 🙂 Expected behavior

After triggering linked editing in vscode:
```tsx
function Test() {
    return <div>
        <T
        <div {...{}}>
        </div>
    </div>
}

```
Linked editing  should not be triggered after `<` in this case
